cmd/root.go

```diff
@@ -120,6 +123,16 @@ func helperFlagFormatter(fs *pflag.FlagSet) string {
 // preExecHook is executed after the command line is parsed but
 // before the command's handler is executed
 func preExecHook(cmd *cobra.Command, args []string) {
+	logLocation, _ := cmd.Flags().GetString("log-loc")
+	_, err := os.Open(logLocation + "/fsoc.log")
+	if err == nil {
+		os.Remove(logLocation + "/fsoc.log")
```

Comment:
consider using https://pkg.go.dev/os#Truncate instead (maybe before open, ignoring not-found)